src/main/java/com/ibm/crypto/plus/provider/ock/NativeInterface.java

```diff
@@ -131,140 +131,50 @@ static void preloadJGskit(String libraryToLoad) {
         osArch = System.getProperty("os.arch");
         String jgskitPath = getJGskitLoadPath();
         File loadFile = null;
-        if (osName.startsWith("Windows")) {
-            if (osArch.equals("x86")) {
-                // win32_x86
-                loadFile = new File(jgskitPath, "libjgskit.dll");
-            } else {
-                // win64_x86
-                loadFile = new File(jgskitPath, "libjgskit_64.dll");
-            }
+        if (osName.startsWith("Windows") && osArch.equals("amd64")) {
```

Comment:
The way the code was before, it would get this dll if OS is `Windows` and anything other than `x86`. Is there anything else architecture-wise that we could get here and want to account for?

Reply:
At the moment i think all we support are windows for 64 bit ( `amd64` arch ). I dont think we support any other windows environments. If the `osName` was to start with `Windows` and we are running on some other architecture then we would get the error on new line 145 just below. Perhaps I am missing a path here that could lead to an error?

Reply:
No, nothing specific comes to mind. I was just wondering if there might be one. But I'll resolve this since you can't think of anything either.